src/main/java/de/uni_potsdam/hpi/asg/common/breeze/model/xml/Components.java

```diff
@@ -51,8 +51,8 @@ public static Components readIn(String filename) {
 			JAXBContext jaxbContext = JAXBContext.newInstance(Components.class);
 			Unmarshaller jaxbUnmarshaller = jaxbContext.createUnmarshaller();
 			if(filename == null || filename.equals("")) {
-				URL url = Components.class.getResource(injarfilename);
```

Comment:
What is the problem with this code? This works for me. However your solution also works. I'm just curious.